Sources/KSCrashFilters/include/KSCrashReportFilterBasic.h

```diff
@@ -56,15 +61,37 @@
  *                    Each "filter" can be id<KSCrashReportFilter> or an NSArray
  *                    of filters (which gets wrapped in a pipeline filter).
  */
-+ (KSCrashReportFilterCombine*) filterWithFiltersAndKeys:(id) firstFilter, ... NS_REQUIRES_NIL_TERMINATION;
++ (instancetype) filterWithFiltersAndKeys:(nullable id) firstFilter, ... NS_REQUIRES_NIL_TERMINATION;
+
+/** Constructor.
+ *
+ * @param filters An array of filters to apply. Each filter should conform to the
+ *                KSCrashReportFilter protocol. If a filter is an NSArray, it will
+ *                be wrapped in a pipeline filter.
+ * @param keys    An array of keys corresponding to each filter. Each key will be
+ *                used to store the output of its respective filter in the final
+ *                report dictionary.
+ */
++ (instancetype) filterWithFilters:(NSArray*) filters keys:(NSArray<NSString*>*) keys;
```

Comment:
Can't. Here is a bad legacy design, where we can pass array of objects or array of arrays. From doc:
```
 * @param filters An array of filters to apply. Each filter should conform to the
 *                KSCrashReportFilter protocol. If a filter is an NSArray, it will
 *                be wrapped in a pipeline filter.
 ```

Reply:
#512